pkg/scheduler/scheduler.go

```diff
@@ -764,6 +762,22 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 	}()
 }
 
+// skipPodSchedule returns true if we could skip scheduling the pod for specified cases.
```

Comment:
We can explain in the comments that two cases we could skip scheduling the pod: 

1. deleting pod; 
2. pod updates could be skipped.

Reply:
Thanks! I make these cases more clear by updating comments(`Case XXX`). Then we do not need to update function doc when we add more cases. Is it OK?

Reply:
Yes. SGTM.